src/frontend/app/features/applications/application-wall/application-wall.component.html

```diff
@@ -11,5 +11,17 @@ <h1>Applications</h1>
 </app-page-header>
 <app-endpoints-missing #appEndpointsMissing></app-endpoints-missing>
 <!-- <p *ngIf="error">We're sorry, we couldn't fetch all of your applications. Please try again later.</p> -->
-<app-list [hidden]="!(endpointsService.haveRegistered$ | async) || !(endpointsService.haveConnected$ | async)">
+
+<ng-template #noEntriesForCurrentFilter>
+  <app-no-content-message [icon]="'apps'" [firstLine]="'There are no apps for the current filter'" [secondLine]="{
+  text: 'Please change the filters or deploy an app that match the current endpoint, organization or space.'
```

Comment:
The filter could be by name. Maybe change to something like `Please change the filters or deploy an application that matches them`